packages/synapse-react-client/mocks/faker/generateDiscussion.ts

```diff
@@ -0,0 +1,40 @@
+import { DiscussionThreadBundle, Forum } from '@sage-bionetworks/synapse-types'
+import { faker } from '@faker-js/faker'
+import { mockUserData } from '../user/mock_user_profile'
+import { pickRandomMockUser } from './fakerUtils'
+import { SetRequired } from 'type-fest'
+
+export function generateForum(
+  overrides: SetRequired<Partial<Forum>, 'projectId'>,
+): Forum {
+  return {
+    id: String(faker.number.int()),
+    etag: faker.string.uuid(),
+    ...overrides,
+  }
+}
+
+export function generateDiscussionThreadBundle(
+  overrides: SetRequired<
+    Partial<DiscussionThreadBundle>,
+    'forumId' | 'projectId'
```

Comment:
When generating a `DiscussionThreadBundle`, require a forumId and projectId. We'll generate everything else.